ci/scripts/python_wheel_windows_build.bat

```diff
@@ -121,7 +127,24 @@ set ARROW_HOME=C:\arrow-dist
 set CMAKE_PREFIX_PATH=C:\arrow-dist
 
 pushd C:\arrow\python
-@REM bundle the msvc runtime
-cp "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Redist\MSVC\14.28.29325\x64\Microsoft.VC142.CRT\msvcp140.dll" pyarrow\
+
+@REM Bundle the C++ runtime
+cp C:\Windows\System32\msvcp140.dll pyarrow\
+
+@REM Build wheel
 python setup.py bdist_wheel || exit /B 1
+
+@REM Repair the wheel with delvewheel
+@REM
+@REM Since we bundled the Arrow C++ libraries ourselves, we only need to
+@REM mangle msvcp140.dll so as to avoid ABI issues when msvcp140.dll is
+@REM required by multiple Python libraries in the same process.
```

Comment:
Can we add to the comment that we are using this remote because of: https://github.com/adang1345/delvewheel/pull/59

```suggestion
@REM required by multiple Python libraries in the same process.
@REM For this we require:  https://github.com/adang1345/delvewheel/pull/59
```

Reply:
I've committed an alternate suggestion, thank you.